File: 04 - JavaScript/05-07-00-While-Sumar5Numeros-ULP.js
// Sumar los primeros 5 numeros (while)
let numero = 1, minimo = 0, maximo = 5, acumuladorSuma = 0
// Ciclo while
while (numero <= maximo){
// Imprimir lo que se va a sumar
    console.log(`${acumuladorSuma} + ${numero}`)
    // Realizamos la suma parcial
    acumuladorSuma += numero
    console.log(acumuladorSuma)
    numero++
}

// Ciclo do while
do{
// Imprimir lo que se va a sumar
    console.log(`${acumuladorSuma} + ${numero}`)
    // Realizamos la suma parcial
    acumuladorSuma += numero
    console.log(acumuladorSuma)
    numero++
}while (numero <= maximo)




// Ciclo do-while
// Reiniciamos las variables
acumuladorSuma = 0
numero = 0
do {
    // Imprimir lo que se va a sumar
    console.log(`${acumuladorSuma} + ${numero}`)
    // Realizamos la suma parcial
    acumuladorSuma += numero
    console.log(acumuladorSuma)
    numero++
} while (numero <= maximo)
console.log(`La suma de los primeros ${maximo} numeros es: ${acumuladorSuma}`)

File: 05 - Python/05-07-00-While-Sumar5Numeros-ULP.py
# Sumar los primeros 5 numeros (while)
numero, minimo, maximo, acumulador_suma = 1, 0, 5, 0
while numero <= maximo:
    print(f'{acumulador_suma} + {numero}')
    acumulador_suma += numero
    print(f'Suma parcial: {acumulador_suma}')
    numero += 1
else:
    print(f'La suma de los primeros {maximo} numeros es: ', acumulador_suma)